### ISSUE TITLE
applyPolyfills() is bundled in core and web components manager

### ISSUE THREAD

**Issue by denyo** [porsche-design-system/porsche-design-system#1023]
Scope
---
Might be obsolete because of #948 

Out of Scope
---
List topics that are not part of this task.

Notes  
---
Depends on #948 

Acceptance criteria
---
- [ ] Criteria

Subtasks
---
- [ ] Task
